.lando.yml

```diff
@@ -30,7 +33,14 @@ services:
 #      - appserver_nginx
 #      - appserver
 
+events:
+  post-start:
+    - appserver: composer install
+    - node: npm install && npm run build
```

Comment:
Thoughts on automatically running these each time the project is started? I see it as a convenience to not have to remember to do these steps in the case that a composer or npm dependency has changed or the theme assets need to be rebuild. However, it does require more time to perform these actions on `lando start`.

Reply:
@dpellenwood Is there any way to preset a prompt for this? `Do you wish to build FE assets? [Y/n]`

Reply:
I'm okay with this each time too